__tests__/app.test.js

```diff
@@ -154,4 +155,47 @@ describe("API Articles", () => {
       })
     })
   })
-});
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("200 return the comments for an article_id, with comment_id, votes, created_at, author, body, article_id. Sorted most recent first", () => {
+      return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then(({body}) => {
+        const firstComment = body.comments[0]
+        if(body.comments.length > 0) {
+          body.comments.map((comment) => {
+            expect(comment).toHaveProperty("comment_id")
+            expect(comment).toHaveProperty("votes")
+            expect(comment).toHaveProperty("created_at")
+            expect(comment).toHaveProperty("author")
+            expect(comment).toHaveProperty("body")
+            expect(comment).toHaveProperty("article_id")
+          })
+          expect(body.comments).toBeSortedBy("created_at", {descending: true})
+          expect(body.comments.length).toBe(11)
+          expect(firstComment.body).toBe("I hate streaming noses")
+          expect(firstComment.article_id).toBe(1)
```

Comment:
Instead of checking only the first comment, I recommend testing all of them in a foreach() (not a map).